### ISSUE TITLE
Action icons in the UI Shell Header do not provide visual text to describe the link purpose

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#7731]
## [Action icons in the UI Shell Header](https://react.carbondesignsystem.com/?path=/story/ui-shell--header-base-w-actions) do not provide visual text to describe the link purpose
This is a failure of [WCAG 2.1 Success Criterion 2.4.4 Link purpose (In context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context#techniques) The intent of this Success Criterion is to help users understand the purpose of each link so they can decide whether they want to follow the link. 
If a user does not know what the icon means, there should be visible text to describe the link's purpose such as a tooltip.

**Comment by tay1orjones** [comment]
@snidersd thanks for reporting this and including the link for context. I've opened a PR #7736 to address it via adding a `title` to the button containing the aria-label text. If I understand correctly, this should meet the success criteria under `#6, H33: Supplementing link text with the title attribute`
